src/runtime-rs/crates/runtimes/common/src/error.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use crate::types::{ProcessID, Response};
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error("failed to find container {0}")]
+    ContainerNotFound(String),
+    #[error("failed to find process {0}")]
+    ProcessNotFound(ProcessID),
+    #[error("failed to translate from response {0} to shim")]
```

Comment:
I think it should be `failed to translate response {0} to shim`. But based on the actual usage in the code, it looks like it's an unexpected response in a given context. So it looks like which context this is in matters. For example, if you can't match a `DeleteResponse`, that matters more than the `from` you got. Could you add the expected response to the error?

Reply:
Thanks for your review, It is more appropriate as an unexpected response and add the expected response.